### ISSUE TITLE
[Investigate?] Discord Spam Messages

### ISSUE THREAD

**Issue by hilyafadhilah** [fastm31/f4stm3#4]
Hi, can anyone confirm how they got here? My Discord account sent links to this repo to everyone, out of nowhere. I didn't go to any malicious website (checked all device history), so I'm still trying to figure out how I got breached. Thanks in advance.

**Comment by Sand5202** [comment]
Wondering if multiple accounts were exposed, a large Discord that I'm in had an account that pasted this repo into multiple channels.

**Comment by EdzkaLT** [comment]
strange that it was created just a few hours ago

**Comment by hilyafadhilah** [comment]
> strange that it was created just a few hours ago

Yeah I noticed too. It's just about the same time the spam attack happened to me.

**Comment by EdzkaLT** [comment]
well i was chatting with that guy in lithuania and he looks like that hes him cuz hes giving me stupid answers and still knows how to do a windows reset.... like is just very strange to me and im pretty sure that this lithuanian guy is behind it.

**Comment by EdzkaLT** [comment]
he said that he downloaded a virus to crack a game but which game he doesnt tell or have any link to it.

**Comment by EdzkaLT** [comment]
and now he turned off his discord this is suspicious to me!

**Comment by EdzkaLT** [comment]
https://igg-games.com/game-dev-tycoon-free-768519945-download.html ok hes back now im sure thats not it can somebody check source ?

**Comment by jrynkiew** [comment]
Just spam. Delete his comment and ban him

**Comment by EdzkaLT** [comment]
youre talkin about me?

> Just spam. Delete his comment and ban him

**Comment by Radek667** [comment]
From what I've seen, when you open the .rar file itself, it executes a code that intercepts your login credentials to discord and other platforms and then sends it to weebhook or email when I opened it in the compiler it was encrypted and there was no way to decrypt it, but I tried, but as you can see it just works by stealing your coding credentials and sending it to you, that's all

**Comment by jrynkiew** [comment]
> youre talkin about me?
> 
> > Just spam. Delete his comment and ban him

Obviously not. I'm talking about the Discord comments this guy posts. Delete them and ban the guy who posts them

**Comment by EdzkaLT** [comment]
> From what I've seen, when you open the .rar file itself, it executes a code that intercepts your login credentials to discord and other platforms and then sends it to weebhook or email when I opened it in the compiler it was encrypted and there was no way to decrypt it, but I tried, but as you can see it just works by stealing your coding credentials and sending it to you, that's all

well can you check any connection between these files https://igg-games.com/game-dev-tycoon-free-768519945-download.html that are presented in here with this discord virus does it have any connection to it ?

**Comment by EdzkaLT** [comment]
> > youre talkin about me?
> > > Just spam. Delete his comment and ban him
> 
> Obviously not. I'm talking about the Discord comments this guy posts. Delete them and ban the guy who posts them

well im chatting with one off the victims, im trying to get info where he got this virus from instead of blocking.

**Comment by SlimyX** [comment]
My friend had a server admin that was info grabbed dm him the same link to that .rar file, he got hold of his account back again but he doesn't know what he did in order to get his account hacked.
